src/scss/06_components/layout/_layout.scss

```diff
@@ -1,4 +1,4 @@
-$-sidebar-width: 340px;
+$-sidebar-width: calc(340px + $spacing-double + $spacing-double);
```

Comment:
Maybe explain from where these `$spacing-double` values come from so that the math can be adjusted if needed at some point.

Reply:
Added comment above to clarify this calc.